tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature

```diff
@@ -62,7 +62,7 @@ Feature: sharing
       | space           | Personal      |
       | sharee          | Carol         |
       | shareType       | user          |
-      | permissionsRole | Viewer        |
+      | permissionsRole | <permissions-role>        |
```

Comment:
this should not be necessary

Reply:
May I please get some clarification(s) why wouldn't it be necessary?

Reply:
share role should not matter for sharer to move/rename the file.